WebRoot/js/zimbraAdmin/statistics/view/ZaGlobalAdvancedStatsPage.js

```diff
@@ -192,7 +192,9 @@ ZaGlobalAdvancedStatsPage.plotGlobalQuickChart = function (id, group, columns, c
                 for (var i = 0; i < timeStampLength; i++) {
                     var valuesSumAtTs = 0;
                     for (var j = 0; j < hostNameLen; j++) {
-                        if (hostname[j].stats[0].values[i] && hostname[j].stats[0].values[i].stat[0]) {
+                        if (hostname[j].stats && hostname[j].stats.length > 0 && 
```

Comment:
To me that doesn't seem very likely to happen. Just because there's a loop till  "hostNameLen" indicating there are elements there in the array unless explicitly some response comes as [value, value , undefined] or sort of which will make me wrong.